src/reporter.js

```diff
@@ -6,6 +6,11 @@ const api = require('./api')
 const debug = require('./debug')
 const shortener = require('./shortener')
 
+const UPDATE_CONFIG_MSG = '⚠️  Please update bundlesize.config.json! '
+const MAX_SIZE_DELTA = 5120 // 5KB off the max size
```

Comment:
Would it be worth to read these from the config file? It didn't seem to me something that would change enough or would need customization enough to deserve it, but we could go as far as to define them independently for each package.